### PR TITLE
documenting how to associate languages, file extensions

### PR DESCRIPTION
**Pull request by anaxceron**
Documenting how to associate a language to a file extension type in VS Code:

- Added question to [ZE FAQs](https://docs.zowe.org/stable/getting-started/zowe_faq#zowe-explorer-faq)
- Added section to [Usage tips](https://docs.zowe.org/stable/user-guide/ze-usage#usage-tips)

Also turned bullet list in [Usage tips](https://docs.zowe.org/stable/user-guide/ze-usage#usage-tips) to a list w/ headers.

### CODE REVIEW

File: docs/user-guide/ze-usage.md [hunk 1]
```diff
@@ -25,21 +25,33 @@ Review this section to familiarize yourself with Zowe Explorer and make the best
...
-Make the best use of the extension with the following tips:
+Make the best use of Zowe Explorer with the following tips:
 
-- **Data sets, USS, and jobs persistence settings:** You can store any data sets, USS files, or jobs permanently in the **Favorites** tab. Right-click on a data set, USS file, or job and click **Add Favorite**.
+### Data sets, USS, and jobs persistence settings
+You can store any data sets, USS files, or jobs permanently in the **Favorites** tab. Right-click on a data set, USS file, or job and click **Add Favorite**.
 
-- **Identify syntax errors with a syntax highlighter:** Zowe Explorer supports a syntax hightlighter for data sets. To enhance the experience of using the extension, you can download an extension that highlights syntax.
+### Identify syntax errors with a syntax highlighter
+Zowe Explorer supports a syntax hightlighter for data sets. To enhance the experience of using the extension, you can download an extension that highlights syntax.
```
Comment (t1m0thyj) [inline docs/user-guide/ze-usage.md:34]: `hightlighter` -> `highlighter`

File: docs/getting-started/zowe_faq.md [hunk 2]
```diff
@@ -423,6 +423,16 @@ Zowe Explorer V2 supports using Service Profiles, Base Profiles, and Team Profil
 
 </details>
 
+### Is it possible to change the detected language of a file or data set opened in Zowe Explorer?
+
+<details className="zowe-faq">
+
+<summary></summary>
+
+Yes, you can configure Visual Studio Code to use a specific language for a particular file extension. To set file associations, see [Add a file extension to a language](https://code.visualstudio.com/docs/languages/overview#_add-a-file-extension-to-a-language).
```
Comment (t1m0thyj) [inline docs/getting-started/zowe_faq.md:432]: Should we mention data sets too? E.g., `for a particular file extension or data set qualifier`

File: docs/user-guide/ze-usage.md [hunk 3]
```diff
@@ -25,21 +25,33 @@ Review this section to familiarize yourself with Zowe Explorer and make the best
...
 
-- **Edit a profile**: You can edit existing profiles listed in the **Side Bar** by clicking the profile's **Edit** icon (next to the **Search** icon). The feature lets you modify the information inside your profile.
+### Change the detected language of a file or data set
 
-- **Delete a profile**: In Zowe V1, you can permanently delete profiles by right-clicking the profile and selecting the **Delete Profile** option. The feature deletes the profile from your `.zowe` folder. In Zowe V2, right-click the profile, and select **Delete Profile** to open the configuration file and manually delete the profile.
+If the language for a file or data set opened in Zowe Explorer is not detected correctly, configure Visual Studio Code to use a specific language for a particular file extension. To set file associations, see [Add a file extension to a language](https://code.visualstudio.com/docs/languages/overview#_add-a-file-extension-to-a-language).
```
Comment (t1m0thyj) [inline docs/user-guide/ze-usage.md:38]: `for a particular file extension` - would `for that file's extension` be clearer?

File: docs/user-guide/ze-usage.md (hunk 1 above)
Comment (anaxceron) [reply docs/user-guide/ze-usage.md:34]: corrected

File: docs/getting-started/zowe_faq.md (hunk 2 above)
Comment (anaxceron) [reply docs/getting-started/zowe_faq.md:432]: done

File: docs/user-guide/ze-usage.md (hunk 3 above)
Comment (anaxceron) [reply docs/user-guide/ze-usage.md]: Updated the text so the scenario applies more broadly. Please review.

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @anaxceron!

**Pull request merged**